src/test/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer has started work.");
```

Comment:
The message 'Bulldozer has started work.' is not very informative. It would be better to indicate that the Bulldozer is working, for example: 'The Bulldozer is now working.'